settings.py

```diff
@@ -1,8 +1,11 @@
-GAME_NAME = 'Frogger City'
+import pygame
 
-SCREEN_WIDTH = 840
-SCREEN_HEIGHT = 640
+GAME_NAME = "Frogger City"
+
+SCREEN_WIDTH = 1280
+SCREEN_HEIGHT = 720
 SCREEN_FPS = 60
+BACKGROUND_IMAGE = pygame.image.load("assets/map_1280x720.png")
```

Comment:
Sollen wir die noch in `map` oder `general_map` oder `main_map` abändern?

Reply:
Filename meinst du?

Reply:
Stimmt. Ich wäre für Beides damit es ganz klar ist.

Reply:
`MAP_IMAGE = pygame.image.load("assets/map.png")`

Reply:
mache ich eben

Reply:
Ich nehme meinen Wunsch zurück :-D

Reply:
Okay wie in whatsapp beschlossen, bleibt doch erst einmal so und wir gucken wenn wir weiter sind noch einmal was mehr Sinn macht.